Assets/Vimeo/Scripts/Recorder/VimeoPublisher.cs

```diff
@@ -114,23 +118,38 @@ private void VideoUpdated(string response)
             UploadProgress("SaveInfoComplete", 1f);
         }
 
+        private void NetworkError(string response)
+        {
+            if (OnUploadFail != null) {
+                OnUploadFail("It seems like you are not connected to the internet, or having connection problems which disables the uploading of the video.");
```

Comment:
This message is technically not accurate. Lots of vimeo api calls are in the Publisher class. Doesnt mean it failed while uploading.

Maybe simply write:
```
It seems like you are not connected to the internet or are having connection problems.
```

Reply:
👌 https://github.com/vimeo/vimeo-unity-sdk/pull/28/commits/0fa27ea469d9ab52374d009097d8c70c475f87d3